uvicorn/importer.py

```diff
@@ -1,13 +1,12 @@
 import importlib
-from types import ModuleType
-from typing import Union
+from typing import Any
 
 
 class ImportFromStringError(Exception):
     pass
 
 
-def import_from_string(import_str: Union[ModuleType, str]) -> ModuleType:
+def import_from_string(import_str: Any) -> Any:
```

Comment:
I understand the current behavior as being wrong but is `Any` the best approach here?

Reply:
I see.

@euri10 Do you have something to say here or can we stick with `Any`?

Reply:
no sounds about the best option with my limited mypy understanding,